infrastructure/modules/xc3/main.tf

```
@@ -206,3 +206,73 @@ resource "aws_lambda_layer_version" "lambda_layer_prometheus" {
     terraform_data.upload_files_on_s3
   ]
 }
+
+## Cloudwatch Alarms
+# tflint-ignore: terraform_required_providers
+resource "aws_cloudwatch_metric_alarm" "this" { 
+  alarm_name          = "${var.namespace}-xc3_cpu_usage_alarm"
```

```suggestion
  alarm_name          = "${var.namespace}-cpu-usage-alarm"
```

Let's not use the xc3 as it will be coming from the workspaces